css/components/misc.css

```diff
@@ -159,6 +159,9 @@
   padding: 0.2em 0.7em;
   border-left: 1px solid #FFFFFF;
 }
+.region-footer .block em {
+  font-family: Georgia, Times, Time New Roman, serif;
+}
```

Comment:
This may be required, but seems to be a very specific implementation for a font face?